contrib/opencensus-context/examples/py27.py

```diff
@@ -0,0 +1,18 @@
+from opencensus.common.runtime_context import RuntimeContext
+
+RuntimeContext.register_slot('correlation_context', lambda: {})
+
+def hello(name):
+    correlation_context = RuntimeContext.correlation_context.copy()
+    correlation_context['name'] = name
+    RuntimeContext.correlation_context = correlation_context
+
+    print(RuntimeContext)
+
+if __name__ == '__main__':
+    print(RuntimeContext)
+    RuntimeContext.correlation_context['test'] = True
+    print(RuntimeContext)
+    hello('hello')
```

Comment:
This might be a more interesting example with a threadpool.

Reply:
Sure, I will add one. Probably also have an example on `__aenter__` and `__aexit__`.

Reply:
Added threadpool example.

Reply:
Added async span example.